README.md

```diff
@@ -356,15 +356,33 @@ Automatically and intelligently transform text, numbers, booleans, and other for
     g = g.umap()  # UMAP, GNNs, use features if already provided, otherwise will compute
 
     # other pydata libraries
-    X = g._node_features
-    y = g._node_target
+    X = g._node_features  # g._get_feature('nodes')
+    y = g._node_target  # g._get_target('nodes')
     from sklearn.ensemble import RandomForestRegressor
     model = RandomForestRegressor().fit(X, y) #assumes train/test split
     new_df = pandas.read_csv(...)
     X_new, _ = g.transform(new_df, None, kind='nodes')
     preds = model.predict(X_new)
     ```
 
+ * Encode model definitions and compare models against each other
+
+   ```python
+    # graphistry
+    from graphistry.features import search_model, topic_model, ngrams_model, ModelDict, default_parameters
+
+    g = graphistry.nodes(df)
+    g = g.umap(X=[..], y=[..], **search_model)
+
+    # set custom encoding model 
```

Comment:
swap in a more compelling & real model, like a different language or for translation?

Reply:
I agree it makes sense, but in this demo the `search_model` (MSMARCO2) is the best model.

Reply:
yep and maybe also a url for where they can find them